### PR TITLE
[Deprecate functionality] Remove loadSafeFile

### PR DESCRIPTION
**Pull request by CristhianMotoche**
# Description
Solves issue #127

# Notes
- Remove references to loadSafeFile
- Update version in cabal file
- Update README

# TODO
- [x] Update CHANGELOG explanation

### CODE REVIEW

File: dotenv.cabal [hunk 1]
```diff
@@ -72,7 +70,6 @@ executable dotenv
                        , process
                        , text
                        , transformers >= 0.4 && < 0.6
-                       , yaml >= 0.8
```
Comment (sestrella) [inline dotenv.cabal:75]: @CristhianMotoche 👏🏼

**Review by sestrella — APPROVED** [review]
@CristhianMotoche LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## MASTER
 
+## Dotenv 0.9.0.0
+* Remove `loadSafeFile`
```
Comment (sestrella) [inline CHANGELOG.md:4]: @CristhianMotoche maybe we could expand a little bit more here about the reasoning behind this change?

**Review by fefi95 — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 ## MASTER
 
+## Dotenv 0.9.0.0
+* Remove `loadSafeFile`. Users must create their own parsers to convert the
```
Comment (sestrella) [inline CHANGELOG.md:4]: @CristhianMotoche 👏🏼

**Pull request merged**